.circleci/config.yml

```diff
@@ -25,6 +25,10 @@ test_containers:
         description: Resource class to use
         type: string
         default: medium
+      edge:
+        description: Use latest version of dependencies during testing
+        type: boolean
+        default: false
```

Comment:
As we discussed, it would be cool if there's some way to have this workflow report to Slack when the build finishes (success or failure) so we can keep an eye on it.